Fixed #32493 -- Removed redundant never_cache uses from admin views. (#14066)
django/django: docs/releases/4.0.txt

  :setting:`STATIC_URL`, the leading slash is removed from that setting (now
  ``'static/'``) in the default :djadmin:`startproject` template.

* The :class:`~django.contrib.admin.AdminSite` method for the admin ``index``

>   view does not now have the ``never_cache`` decorator applied if accessed

Reviewer: ```suggestion
  view is no longer decorated with ``never_cache`` when accessed
```
